api/src/identity-access-management/domain/emails/create-warning-connection.email.js

```diff
@@ -0,0 +1,51 @@
+import { LOCALE } from '../../../shared/domain/constants.js';
+import { urlBuilder } from '../../../shared/infrastructure/utils/url-builder.js';
+import { EmailFactory } from '../../../shared/mail/domain/models/EmailFactory.js';
+import { mailer } from '../../../shared/mail/infrastructure/services/mailer.js';
+
+/**
+ * Creates an email to warn users for suspicious connection.
+ *
+ * @param {Object} params - The parameters for creating the email.
+ * @param {string} params.locale - The locale for the email.
+ * @param {string} params.email - The recipient's email address.
+ * @param {string} params.firstName - The recipient's first name.
+ * @returns {Email} The email object.
+ */
+export function createWarningConnectionEmail({ locale, email, firstName}) {
+  const factory = new EmailFactory({ app: 'pix-app', locale });
+
+  const { i18n, defaultVariables } = factory;
+  const pixAppUrl = urlBuilder.getPixAppBaseUrl(locale);
+  const resetUrl =
+    locale === LOCALE.FRENCH_FRANCE
+      ? `${pixAppUrl}/mot-de-passe-oublie`
+      : `${pixAppUrl}/mot-de-passe-oublie?lang=${locale}`;
```

Comment:
Est ce qu'on ne peut pas simplement écrire ?
```suggestion
  const resetUrl = ${pixAppUrl}/mot-de-passe-oublie?lang=${locale}`;
```

Reply:
@bpetetot On peut, mais ça donnerait une addresse avec un domaine en  ".fr" avec "?lang=fr", ce n'est pas étrange ? 🤔

Reply:
Vu ensemble, ce n'est pas grave :)